test/cppsuite/tests/run.cpp

```diff
@@ -271,7 +274,7 @@ main(int argc, char *argv[])
                 else
                     current_cfg = cfg;
 
-                error_code = run_test(current_test_name, current_cfg, wt_open_config, home);
+                error_code = run_test(current_test_name, current_cfg, wt_open_config, home, true);
```

Comment:
Can we simply do:
```
if (current_test_name == api_instruction_count_benchmarks)
    // Log something to say we skip this one
else
    error_code = run_test(current_test_name, current_cfg, wt_open_config, home);
```

Reply:
[f4f9d27](https://github.com/wiredtiger/wiredtiger/pull/10602/commits/f4f9d274fc894af1382ec898399069513eed204b)